transports/alltransports/alltransports.go

```diff
@@ -10,6 +10,7 @@ import (
 	_ "github.com/containers/image/docker"
 	_ "github.com/containers/image/docker/archive"
 	_ "github.com/containers/image/docker/daemon"
+	_ "github.com/containers/image/oci/archive"
```

Comment:
Can this have a smoke test with the others?

Reply:
… this is also outstanding, not blocking I guess.

Reply:
I added tests for oci_transport, are there any other tests that I am missing?

Reply:
`transports/alltransports/alltransports_test.go:TestImageNameHandling`

Reply:
added the test. @mtrmac PTAL